runtime/bindings/python/invoke.cc

```diff
@@ -452,8 +452,7 @@ class InvokeStatics {
         sub_packer(c, item_list.raw_ptr(), py_item);
       }
       // Push the sub list.
-      iree_vm_ref_t retained =
-          iree_vm_list_retain_ref(item_list.steal_raw_ptr());
+      iree_vm_ref_t retained = iree_vm_list_move_ref(item_list.steal_raw_ptr());
```

Comment:
Should this still be called "retained"?

Reply:
Yes, it is still "retaining" its reference. It is just the reference that it stole. The intent with the naming is that it is obvious that someone needs to transfer this or release it.